src/modules/ecdsa_adaptor/tests_impl.h

```diff
@@ -1105,12 +1105,6 @@ void adaptor_tests(void) {
         secp256k1_ecdsa_signature sig_tmp;
```

Comment:
`sig_tmp` is no longer used with the removal of the test at 1122.

Reply:
fixed